### ISSUE TITLE
Shadows of infecteds visibles to survivors through ceilings and walls

### ISSUE THREAD

**Issue by lunatixxx** [Tsuey/L4D2-Community-Update#114]
Some places in certain maps will allow that, dark carnival map 2 ladder ceiling before luge being a good example. 
If we could determine the shadow of infecteds i guess it could be fixable with code like that, but right now all shadows are only one entity?

https://github.com/SirPlease/L4D2-Competitive-Rework/blob/934bb47cda97601cc80ee7fac40f3eacf6ec6674/addons/sourcemod/scripting/l4d2_blind_infected.sp#L165

**Comment by Derpduck** [comment]
Just to note, the plugin linked in the OP is unrelated.
This is "fixed" in Zonemod by completely disabling shadow_control, which disables all dynamic shadows, not really an ideal solution.

https://github.com/SirPlease/L4D2-Competitive-Rework/blob/master/addons/sourcemod/scripting/l4d2_shadow_removal.sp

**Comment by alexiscoutinho** [comment]
Yeah it's from a custom campaign, but if someone else reading this wants a video example, here it is https://youtu.be/g_WAJpaEkjs?t=324.
